lib/api/unittest/CPersistenceManagerTest.cc

```diff
@@ -3,7 +3,8 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-#include "CBackgroundPersisterTest.h"
+#include "../../../include/api/CPersistenceManager.h"
```

Comment:
Can this be just `#include <api/CPersistenceManager.h>` and grouped with the other `api` library includes?